README.md

```diff
@@ -1,14 +1,144 @@
 # Sparkle Web App
 
-Codebase for the Sparkle Platform, brought to you by [Sparkle](https://sparklespaces.com/).
+Codebase for the Sparkle platform, brought to you by [Sparkle](https://sparklespaces.com/), a platform for the most immersive & interactive virtual events.
 
-## Getting started
 
-The instructions below assume you already have a Firebase project setup and configured appropriately.
+### Getting started
 
-**Note**: If you're interested in setting up the project to run in your own environment but you're not sure how to go about it, feel free to [open an issue](https://github.com/sparkletown/sparkle/issues/new) asking for assistance.
+What you will need:
 
-### Frontend
+- A free Firebase account
+- Your favorite command-line interface (e.g. Terminal or zsh on a Mac)
+- Optional: accounts with Stripe, Twillio
+- A little fairy dust
+
+---
+<!-- section 1 -->
+
+## Part One: Firebase Project Setup
+
+### Step 1: Create New Firebase Project
+
+1. Go to https://console.firebase.google.com/u/2/.
+
+2. Click _Add project_.
+  * Fill out the name field with something like `example-project`
+  * Click _Continue_
+
+
+3. Optional: enable Google Analytics.
+
+  * Click _Continue_
+  * Configure Google Analytics
+  * Click _Create project_
+  * Wait for project to be created
+  * Click _Continue_
+
+This part of the setup is complete!
+
+
+### Step 2: Configure Firebase Project Settings
+
+1. Go to https://console.firebase.google.com/ and find the `Example Project` you chose in step 1.
+
+2. From _Project Overview_, hover over the gear icon and click _Project Settings_.
+
+ * In the 'General' tab:
+
+    * Set the public-facing name.
+   * In the ['Users & Permissions'](https://console.firebase.google.com/u/2/project/`example-project`/settings/iam) tab:
+
+    * Add any other users who will need "Owner" level access to this project (e.g. a backup owner who can change things if needed, deployments).
+    * Add the users who will need "Editor" level access to this project.
+
+3. Optionally, upgrade your account. Click the gear icon next to _Project Overview_ again, and then select ['Usage and billing'](https://console.firebase.google.com/u/2/project/`example-project`/usage), then select the Blaze plan.
```

Comment:
Don't create links to the admin console like this, they won't work, and needlessly complicates things. Just describe what to do.